### PR TITLE
[mssql-linux] Added feature to enable HADR

### PR DESCRIPTION
**Pull request by thomasliddledba**
Adding feature to enable SQL Server deployment with HADR (Availability Groups)

### CODE REVIEW

**Comment by thomasliddledba** [comment]
Fixed merge conflicts #5291

File: stable/mssql-linux/README.md [hunk 1]
```diff
@@ -98,8 +98,8 @@ The configuration parameters in this section control the resources requested and
 | service.port     | Service Port                                                                                   | `1433`                           |
 | collation        | Default collation for SQL Server                                                               | `SQL_Latin1_General_CP1_CI_AS`   |
 | lcid             | Default languages for SQL Server                                                               | `1033`                           |
-| persistence.enabled | Persist the Data and Log files for SQL Server                                               | `false`                          |
-| persistence.enabled | Persist the Data and Log files for SQL Server                                               | `false`                          |
+| hadr             | Enable Availability Group                                                                      | `0`                              |
+| persistence.enabled | Presist the Data and Log files for SQL Server                                               | `false`                          |
```
Comment (unguiculus) [inline stable/mssql-linux/README.md]: Revert `Presist` to `Persist`

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/mssql-linux/README.md (hunk 1 above)
Comment (thomasliddledba) [reply stable/mssql-linux/README.md]: Fat finger - Fixed.

**Review by timstoop — APPROVED** [review]

**Pull request merged**